tests/spread/general/plugin-python-3.6/rockcraft.orig.yaml

```diff
@@ -0,0 +1,23 @@
+name: plugin-python-36
+base: placeholder-base
+build-base: ubuntu:20.04
+version: '0.1'
+summary: A project using Python 3.6
+description: A project using Python 3.6
+license: GPL-3.0
+platforms:
+    amd64:
+
+package-repositories:
+  - type: apt
+    ppa: deadsnakes/ppa
+    priority: always
+
+parts:
+    my-part:
+        plugin: python
+        source: .
+        stage-packages: [python3.6-venv]
+        python-packages: [black]
+        build-environment:
+            - PARTS_PYTHON_INTERPRETER: python3.6
```

Comment:
What happens if the user tries to specify the full path for the interpreter? If that's not allowed, maybe we should detect and warn? Or, if that's valid, the shebang line shouldn't hardcode `/bin/`.

Reply:
Hmm good question. This `PARTS_PYTHON_INTERPRETER` variable is used both during build and during runtime (in the generated containers) and I'm having a hard time thinking of a full path value that would work in both "stages", so I think we should block it. I'll add a check and a test

Reply:
Added check & test